packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -152,69 +196,13 @@ export default class extends DataAggregatingRouteHandler {
         }
       }),
     );
-
-    // start fetching options
-    const optionsTasks = overlays.map(o => this.fetchMeasureOptions(o, this.query));
```

Comment:
Rename existed function `fetchMeasureOptions` to `fetchMeasureOption`. Remove `this.query` because it is unused.